.merlin

```diff
@@ -20,6 +20,7 @@ B _build/lib/bap_plugins
 B _build/lib/text_tags
 B _build/lib/regular
 B _build/lib/graphlib
+B _build/lib/microx
```

Comment:
Ah. Then it should go in `propagate_taint` .merlin, since that is where I wasn't resolving it. https://github.com/BinaryAnalysisPlatform/bap/blob/master/plugins/propagate_taint/.merlin